### PR TITLE
Add templates and sample stub files for aws and warden

### PR DESCRIPTION
**Pull request by yuanzhao**
This was modelled after [cf-release](https://github.com/cloudfoundry/cf-release/).

### CODE REVIEW

File: templates/logsearch-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  max_errors: 1
+
+resource_pools: 
+- name: logsearch
+  network: default
+  stemcell: (( meta.stemcell ))
+  cloud_properties: (( merge ))
+  env: (( merge || meta.default_env ))
+
+- name: logsearch_api
```
Comment (dpb587) [inline templates/logsearch-deployment.yml:44]: Out of curiosity, is there background or a reason for the resource pools all being prefixed with `logsearch`? I thought resource pools were deployment-specific already in which case it seems redundant; but if this sort of thing is good practice I'm interested to learn.

File: jobs/ingestor_relp/spec [hunk 2]
```diff
@@ -24,6 +24,7 @@ properties:
 
   logstash_ingestor.relp.port: 
     description: Port to listen for RELP messages
+    default: 2514
```
Comment (dpb587) [inline jobs/ingestor_relp/spec:27]: Next time, please at least make this sort of change a separate commit. It's very unrelated to the commit message for people reviewing logs to see what changes were happening.

**Comment by dpb587** [comment]
I'm don't really use spiff myself, but I was able to test out the warden setup and these templates will definitely make it more welcoming to others. Thanks for writing them up.

Merged.

File: templates/logsearch-deployment.yml (hunk 1 above)
Comment (jim80net) [inline templates/logsearch-deployment.yml:44]: No reasoning, though [one day](https://github.com/cloudfoundry/bosh-notes/blob/master/cloud-config.md) BOSH will manage resource pools. Then ours will be to consume, not create. =]
